test/kokoro/xds_k8s_lb.sh

```diff
@@ -167,8 +167,10 @@ main() {
     "outlier_detection_test"
     "remove_neg_test"
     "round_robin_test"
-    "bootstrap_generator_test"
   )
+  if [ "${TESTING_VERSION}" != "master" ]; then
```

Comment:
We need the opposite: only add it when the version IS master

Reply:
Whoops, sorry, missed that.